### PR TITLE
Feature/#505 add quiz fsm

### PR DESCRIPTION
**Pull request by ritik307**
Added Pop Quiz in the `docs/seq-design`

Fixes #505 

#### Changes done:
- [x] Pop Quiz in `fsm.md`

#### Screenshots
FSM

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by ritik307** [comment]
@Shivansh2407  plz review this PR

File: docs/seq-design/fsm.md [hunk 1]
```diff
@@ -101,3 +101,22 @@ Now translate the options which are listed above into an FSM diagram/flow-chart:
...
+   1. State diagram
+   * Block diagram
+   * Flow diagram
+   * All of these
+2. Which of the following is a type of `FSM(Finite State Machine)`?
+   1. Mealy State Machine
+   * Moore State Machine
+   * Both 
+   * None   
+3. Finite State Machine is said to be `Mealy state machine`?
```
Comment (ayan-biswas0412) [inline docs/seq-design/fsm.md:117]:
```suggestion
3. In `Mealy state machine` the outputs depends on ?
```

File: docs/seq-design/fsm.md [hunk 2]
```diff
@@ -101,3 +101,22 @@ Now translate the options which are listed above into an FSM diagram/flow-chart:
...
+   1. Outputs depend on present inputs & present states.
```
Comment (ayan-biswas0412) [inline docs/seq-design/fsm.md:118]:
```suggestion
   1. on present inputs & present states.
```

File: docs/seq-design/fsm.md [hunk 3]
```diff
@@ -101,3 +101,22 @@ Now translate the options which are listed above into an FSM diagram/flow-chart:
...
+   * Outputs depend only on present states.
```
Comment (ayan-biswas0412) [inline docs/seq-design/fsm.md:119]:
```suggestion
   * Only on present states.  
```

File: docs/seq-design/fsm.md [hunk 4]
```diff
@@ -101,3 +101,22 @@ Now translate the options which are listed above into an FSM diagram/flow-chart:
...
+   * All of these
+2. Which of the following is a type of `FSM(Finite State Machine)`?
+   1. Mealy State Machine
+   * Moore State Machine
+   * Both 
+   * None   
+3. Finite State Machine is said to be `Mealy state machine`?
+   1. Outputs depend on present inputs & present states.
+   * Outputs depend only on present states.  
+3. Finite State Machine is said to be `Moore state machine`?
```
Comment (ayan-biswas0412) [inline docs/seq-design/fsm.md:120]: make same changes Here also

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
bigger options with repeated strings I think I doesn't give a good visual aspect rather try to give some smaller options keeping the meaning same and I have also mentioned some of changes please change the question no 4 also in similar way and please insert another question such that Total number becomes 5

**Comment by ritik307** [comment]
@ayan-biswas0412 thanks for reviewing the code will do all the mentioned changes ASAP.

**Comment by Shivansh2407** [comment]
> 
> 
> @ayan-biswas0412 thanks for reviewing the code will do all the mentioned changes ASAP.

@ritik307 could you please make the specified changes so that we can merge this PR. Thanks a lot!

**Comment by ritik307** [comment]
@Shivansh2407 Done! please review

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
remove unnecessary spaces from the files in which you have changed nothing

**Comment by manjotsidhu** [comment]
Closing this Pull Request since it's not active. You can make a new Pull Request anytime with the changes. Thanks for the contribution.

**Pull request closed without merging**